chart/localstack/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
+{{- if .Values.ingress.enabled }}
+{{- range $host := .Values.ingress.hosts }}
+  {{- range $.Values.ingress.paths }}
```

Comment:
Error: template: templates/NOTES.txt:4:12: executing "templates/NOTES.txt" at <.paths>: can't evaluate field paths in type interface {}

values.yml - ingress
ingress:
  enabled: true
  annotations: {}
    # kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"
  paths:
    - /
  hosts:
    - chart-example.local
  tls: []
  #  - secretName: chart-example-tls
  #    hosts:
  #      - chart-example.local

Reply:
The error is there also when paths isn't altered from the PR